python_modules/dagster/dagster/_utils/cached_method.py

```diff
@@ -13,7 +13,10 @@
 
 NO_ARGS_HASH_VALUE = 0
 
+# pydantic fields require _ prefix
 CACHED_METHOD_CACHE_FIELD = "_cached_method_cache__internal__"
+# namedtuple does not allow _ prefix
+ALT_CACHED_METHOD_CACHE_FIELD = "cached_method_cache__internal__"
```

Comment:
lol classic

Reply:
If this going to pollute typeaheads?

If so i think we should use the prefix `zcache` in order to hide it

Reply:
`ze_cache` for the french speakers

Reply:
`zee_cache` for the 🇩🇪 homies